### DISCUSSION TITLE
Loading file from resource folder

### DISCUSSION THREAD

**Discussion by AnkS4** [beeware/briefcase#798] [General]
I need to specify the path to one of the file.

I did it via:
`path = Path(Path.cwd(), "src/speechtotext/resources/....")`

It works with the
`briefcase dev`

But, after running an AppImage, it returns
`Folder '/home/anks/beeware/speechtotext/linux/src/speechtotext/resources/...' does not contain model files. Make sure you specified the model path properly in Model constructor. If you are not sure about relative path, use absolute path specification.`

How do I define the absolute/relative path loading a file from resources folder for AppImage?

**Comment by freakboy3742** [comment]
This is because `Path.cwd()` is not a reliable path - depending on the way the app is started, it could be a different location, or not available at all. 

Instead, you should use the `.paths` attribute on your Toga App object. That object has a number of attributes that are reliable path anchors - `paths.app` is the location of your app module, for example.
